components/dashboard/src/teams/OrgSettingsPage.tsx

```diff
@@ -84,7 +84,7 @@ function getTeamSettingsMenu(params: {
     }
     if (orgGitAuthProviders) {
         result.push({
-            title: "Git Integrations",
+            title: "Git Providers",
```

Comment:
**thought**: It feels like the organization part belongs to a broader settings area that could be called Security, Authentication, etc. and the user part is more like an Integration or a Login Connection. However, this should become clearer to everyone once these product areas grow ith more options, etc.